django/thunderstore/core/api_urls.py

```diff
@@ -1,5 +1,6 @@
 from django.urls import include, path
 
+from thunderstore.api.urls import urls as cyberstorm_urls
```

Comment:
Instead of exporting `urls` and renaming it to `cyberstorm_urls` in the import side here, how about we just export `cyberstorm_urls` from the get-go? It's a fair assumption we might have non-cyberstorm URL definitions in the `thunderstore.api.urls` module, so treating the entire module as only cyberstorm-urls is not ideal.

This is a non-blocker but mentioning anyway